walkthroughs/week-4-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,13 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+@Test
+public void testGreetingTrimsWhitespace() {
+  Greeter greeter = new Greeter();
+
+  String greeting = greeter.greet("   Ada   ");
+
+  // Whitespace should be trimmed
+  Assert.assertEquals("Hello Ada", greeting);
+}
```

Comment:
Please add a space before here and indent this properly. You can use `clang-format` as detailed in the [Cheatsheet](https://docs.google.com/document/d/1yeDuDfEoVh2RB2wC-KrhkXNlekc15RxkPy8p-G9U4tA/edit#heading=h.el4nfk7uwuhr) to fix this. E.g. :
```shell
clang-format -i --style=Google walkthroughs/week-4-tdd/intro/src/test/java/com/google/sps/*
```